docs/Reference/classes/Array/deferred_layout.md

```diff
@@ -1,7 +1,7 @@
 [//]: # (Project: Accera)
-[//]: # (Version: 1.2.0)
+[//]: # (Version: 1.2)
 
-# Accera 1.2.0 Reference
+# Accera 1.2 Reference
```

Comment:
Do we need to keep the version in the section title? The doc version is more or less tied to the code version just because they're in the same repo

Reply:
hmm, I was hoping not to maintain it so often. On 2nd thought I can have a script that checks the tag version and update this at doc build time.